upgrade/steps_helpers.go

```diff
@@ -511,18 +511,25 @@ var getExpectedTargetLatest = stepv2.Func21E("From Upstream Releases", func(ctx
 	name, upstreamOrg string) (*UpstreamUpgradeTarget, error) {
 	latest := stepv2.Cmd(ctx, "gh", "release", "list",
 		"--repo="+upstreamOrg+"/"+GetContext(ctx).UpstreamProviderName,
-		"--limit=1",
 		"--exclude-drafts",
 		"--exclude-pre-releases")
-
-	tok := strings.Fields(latest)
-	contract.Assertf(len(tok) > 0, fmt.Sprintf("no releases found in %s/%s",
-		upstreamOrg, GetContext(ctx).UpstreamProviderName))
-	v, err := semver.NewVersion(tok[0])
-	if err != nil {
-		return nil, err
+	
+	versions := strings.Split(latest, "\n")
+	for _, ver := range versions {
+		tok := strings.Fields(ver)
+		contract.Assertf(len(tok) > 0, fmt.Sprintf("no releases found in %s/%s",
+			upstreamOrg, GetContext(ctx).UpstreamProviderName))
```

Comment:
The `f` in `contract.Assertf` means that it acts like `fmt.Sprintf` for its inputs. This is a pretty common go idiom.
```suggestion
		contract.Assertf(len(tok) > 0, "no releases found in %s/%s",
			upstreamOrg, GetContext(ctx).UpstreamProviderName)
```